deploy/updater/updater.py

```
@@ -9,324 +9,257 @@
+
+async def get_cache_data(mc, key_b, default_response={}):
+    cache = await mc.get(key_b)
+
+    if cache:
+        cache = json.loads(cache.decode("utf-8"))
+    else:
+        cache = default_response
+
+    return cache
+
+async def get_alerts(mc, key_b, default_response={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)